src/app/data-planes/views/DataPlaneDetailView.vue

```diff
@@ -240,34 +240,38 @@
                     Non mesh traffic
                   </ServiceTrafficCard>
                 </ServiceTrafficGroup>
-                <ServiceTrafficGroup
-                  type="outbound"
+                <template
+                  v-if="traffic.outbounds.length > 0"
```

Comment:
Something we spotted the other day: This stops an empty "group" (the colored bit) from showing if there are no outbounds, I figured the inbounds doesn't need it as it doesn't have a colored wrapper.